### PR TITLE
Fixing broken link

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-442.surge.sh

Updated pages:
- [Troubleshooting](https://neo4j-docs-aura-442.surge.sh/aura/tutorials/troubleshooting)

File: modules/ROOT/pages/tutorials/troubleshooting.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 This page provides possible solutions to several common issues you may encounter when using Neo4j Aura.
 
-Regardless of the issue, viewing the link:/docs/aura/platform/logging/[Aura query log] is always recommended to monitor processes and verify any problems.
+Regardless of the issue, viewing the link:/docs/aura/platform/logging/download-logs/[Aura query log] is always recommended to monitor processes and verify any problems.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/tutorials/troubleshooting.adoc:7]:
```suggestion
Regardless of the issue, viewing the xref:platform/logging/download-logs.adoc[Aura query log] is always recommended to monitor processes and verify any problems.
```

**Review by AlexicaWright — COMMENTED** [review]
The link didn't work, I think my suggestion does.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-442.surge.sh

Updated pages:
- [Troubleshooting](https://neo4j-docs-aura-442.surge.sh/aura/tutorials/troubleshooting)

**Review by AlexicaWright — APPROVED** [review]
LGTM!

**Pull request merged**